src/options.ts

```diff
@@ -191,6 +191,12 @@ export interface IProps<FlatListItem = any, SectionListItem = any> {
    */
   sectionListProps?: Animated.AnimatedProps<SectionListProps<SectionListItem>>;
 
+  /*
+   * Menu component that will overlay Modalize completely, independent of scrolling.
+    It requires zIndex child with absolute positioning
+   */
+  menuOverlay?: any;
```

Comment:
Could we rename it to?

```suggestion
  FloatingComponent?: ReactNode;
```

First, to follow the same convention with `HeaderComponent`/`FooterComponent` and second to be more specific about the behavior. I'm still not convinced about `Floating` though, because if you don't pass position:absolute/z-index it won't work. Maybe `SeparateComponent`/`AboveComponent` 🤔?

Reply:
Yes, that's better I couldn't find a decent name for it.